contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+
+- Developing a growth strategy, experiment framework, and measurement for both.
+- Doing SEO optimization on the marketing site. Building landing pages. Optimizing the signup funnels.
+- Running A/B tests on specific parts of the signup and activation funnel. Trying to improve the conversion rate for both.
+- Setting up in-app and push notification messaging systems. Building tools and systems for better product onboarding.
+
+### What do PostHog’s growth engineers work on?
+
+To give a specific example of what growth teams work on, we can look at our [growth team at PostHog](/handbook/small-teams/growth). Over the past few months, this team focused on three areas:
+
+1. **Billing.** Creating a [billing service](https://github.com/PostHog/posthog/pull/11986) enabling different pricing for different plans and [products](https://github.com/PostHog/posthog/pull/15035/files). Improving the billing page both on the [website](https://github.com/PostHog/posthog.com/pull/5181) and [in-app](https://github.com/PostHog/posthog/pull/14652) to work with this service.
+2. **Signup.** Improving the signup funnel. Running A/B tests of options like [single sign on buttons](https://github.com/PostHog/posthog/pull/13234). [Verifying emails](https://github.com/PostHog/posthog/pull/13632) and adding two-factor authentication.
+3. **Onboarding.** Setting up [in-app messaging](https://github.com/PostHog/posthog/pull/13108) and notifications. Set up customer.io. Add [autocapture](https://github.com/PostHog/posthog/pull/14687) and session recording toggles.
+
+Billing is a major focus recently because we believe we can provide a better offering to users if we charge for separate products. This will lead to grow across the signup, usage, and revenue metrics the growth team cares about. This also means implementing this is on the growth team’s plate, and it’s a big task.
+
+## Why do growth engineers matter?
+
+Many businesses and engineering teams lose focus on the core metrics needed to grow. Growth engineering work across teams, products, and code base to keep this focus. In doing so, they can make a massive impact on those metrics.
+
+There are many growth opportunities a standard engineering team doesn’t care about, but growth engineers do. This is because those engineering teams don’t set their goals as driving signups or activation. They focus on making the product better, which is necessary for growth engineers to exist. You can’t growth a non-existent product. 
+
+There are optimizations to the signup and onboarding flow that don’t fall under the responsibility of any of our product teams. These are opportunities for the growth team. For example, making all the changes to billing to treat pricing as a product is a lot of work, but unlocks a lot of growth. If we didn’t have a growth team, we’d likely need a team dedicated to billing anyways. 
+
+### What are the results of growth engineers at PostHog?
```

Comment:
Maybe good to include examples of results/objectives/etc. from growth teams outside of PostHog?